Tests/CorruptCore/Serialization/Resources/DefaultStockpile.json

```diff
@@ -1,8 +1,7 @@
 {
-  "StashKeys": [],
-  "Filename": null,
-  "ShortFilename": null,
   "RtcVersion": null,
   "VanguardImplementation": null,
-  "MissingLimiter": false
+  "MissingLimiter": false,
+  "StashKeys": [],
+  "Filename": null
```

Comment:
Note: this is how `Newtonsoft.Json` handles property serialization. It appears to treat properties just like fields, but the grouping / ordering is different. Either way, this should break compatibility.